index.html

```diff
@@ -97,14 +97,14 @@ <h3>Standard</h3>
                     <li><label class="checkbox" title="Attribute name must be lowercase."><input type="checkbox" id="attr-lowercase"> attr-lowercase</label></li>
                     <li><label class="checkbox" title="Attribute value must closed by double quotes."><input type="checkbox" id="attr-value-double-quotes"> attr-value-double-quotes</label></li>
                     <li><label class="checkbox" title="Attribute must set value."><input type="checkbox" id="attr-value-not-empty"> attr-value-not-empty</label></li>
-                    <li><label class="checkbox" title="Attribute name can not been duplication."><input type="checkbox" id="attr-no-duplication"> attr-no-duplication</label></li>
+                    <li><label class="checkbox" title="Attribute name can not been duplicated."><input type="checkbox" id="attr-no-duplication"> attr-no-duplication</label></li>
```

Comment:
This should say "be duplicated" instead of "been duplicated".

But perhaps a different wording would work even better:

```suggestion
                    <li><label class="checkbox" title="Attribute name can not occur more than once."><input type="checkbox" id="attr-no-duplication"> attr-no-duplication</label></li>
```

(That's because "duplicate" implies two copies, while this violation is true of any number of replicas).

Reply:
'can not occur more than once' - great suggestion - that's better.